### DISCUSSION TITLE
Google

### DISCUSSION THREAD

**Discussion by mustafa-archived** [catppuccin/catppuccin#1913] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

Stylus extension firefox

### Has the requested port been themed already?

yes, but still in WIP https://github.com/mustafakhalaf-git/google

**Comment by sgoudham** [comment]
Hi, could you please update the existing discussion #1837 with a comment saying that you are also interested in theming it, thanks!
